src/providers/ruby.rs

```diff
@@ -256,6 +261,14 @@ impl RubyProvider {
                 .contains("Rails::Application")
     }
 
+    fn uses_sprockets(&self, app: &App) -> Result<bool> {
+        if app.includes_file("Gemfile") {
+            let gemfile = app.read_file("Gemfile").unwrap_or_default();
+            return Ok(gemfile.contains("sprockets"));
```

Comment:
[Propshaft](https://github.com/rails/propshaft) also requires an `assets:precompile` step

Reply:
Nice catch, thank you! Added in 291e19aba7b224402a5a698c745e3c891ba4f8f6.